.github/workflows/test-backend.yml

```diff
@@ -6,9 +6,20 @@ jobs:
   test:
     runs-on: ubuntu-latest
 
-    # FIXME: Add PostgreSQL as a dependency to test database connection
-    # services:
-    #   postgres:
+    services:
+      db:
+        image: postgres:16-alpine
+        env:
+          POSTGRES_DB: postgres
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+        options: >-
```

Comment:
This option prevents this workflow from running tests without healthy database, it's unlikely to happen tho